### PR TITLE
xplat: Installation script

### PR DESCRIPTION
**Pull request by obastemur**
Also moved some other helper scripts under tools folder.

### CODE REVIEW

**Comment by obastemur** [comment]
To test this script file, copy-paste command below (root perm. is NOT needed)

```
curl -SL https://aka.ms/chakracore/install | bash
```

Thanks!

**Comment by obastemur** [comment]
Disclaimer/Warning: The script and the binary package it downloads are experimental. If you want to test, do it on your own risk.

**Comment by rmrewak** [comment]
@obastemur script works for me. Tested with Ubuntu 14.02 lts, OSX 10.12, Fedora 24

**Comment by obastemur** [comment]
ping @jianchun

File: tools/XPlatInstall/install_ch.sh [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+ERROR_COLOR='\033[0;31m'
+SUCCESS_COLOR='\033[0;32m'
+CURRENT_OS=
+CHAKRA_VERSION=0
+BINARY_NAME=
+
+PRINT()
+{
+    TEXT_COLORED="$1$2"
+    echo -e "${TEXT_COLORED} ${NORMAL_COLOR}"
```
Comment (jianchun) [inline tools/XPlatInstall/install_ch.sh]: >NORMAL_COLOR [](start = 31, length = 12)

Where is NORMAL_COLOR defined?

**Comment by jianchun** [comment]
:shipit:

**Comment by obastemur** [comment]
@dotnet-bot test Ubuntu ubuntu_linux_debug_static please

**Comment by obastemur** [comment]
@jianchun Thanks for review.

**Pull request merged**